src/backend/marsha/core/tests/api/playlists/test_claim.py

```diff
@@ -0,0 +1,116 @@
+"""Tests for the Playlist claim API of the Marsha project."""
+
+from django.test import TestCase
+
+from marsha.core import factories
+from marsha.core.models import ADMINISTRATOR, INSTRUCTOR, PlaylistAccess
+from marsha.core.simple_jwt.factories import UserAccessTokenFactory
+
+
+class PlaylistClaimAPITest(TestCase):
+    """Test the claim API for playlist objects."""
+
+    maxDiff = None
+
+    def test_claim_playlist_by_anonymous_user(self):
+        """Anonymous users cannot claim playlists."""
+        playlist = factories.PlaylistFactory()
+        response = self.client.post(
+            f"/api/playlists/{playlist.id}/claim/",
+            content_type="application/json",
+        )
+
+        self.assertEqual(response.status_code, 401)
+        playlist.refresh_from_db()
+        self.assertFalse(PlaylistAccess.objects.filter(playlist=playlist).exists())
+
+    def test_claim_playlist_by_random_logged_in_user(self):
+        """Random logged-in users cannot claim playlists unrelated to them."""
+        user = factories.UserFactory()
+        playlist = factories.PlaylistFactory()
+
+        jwt_token = UserAccessTokenFactory(user=user)
+
+        response = self.client.post(
+            f"/api/playlists/{playlist.id}/claim/",
+            HTTP_AUTHORIZATION=f"Bearer {jwt_token}",
+            content_type="application/json",
+        )
+        self.assertEqual(response.status_code, 403)
+        playlist.refresh_from_db()
+        self.assertFalse(PlaylistAccess.objects.filter(playlist=playlist).exists())
+
+    def test_claim_playlist_by_orga_administrator(self):
+        """Organization administrators can claim playlists."""
```

Comment:
can or can't ? Because in the test the status_code is `403`.
And why an organization admin should not be able to claim a playlist ?